test/e2e/testcafe-runner.js

```diff
@@ -0,0 +1,38 @@
+var path           = require('path');

```

Comment:
Just use command line, like shown in this recipe: http://devexpress.github.io/testcafe/documentation/recipes/test-static-html-pages.html